Update al.m3u

eurohunter opened the pull request:
Removed non-working channels and updated links to some other channels

eurohunter committed:
Update al.m3u

Removed non-working channels and updated links to some other channels

freearhey reviewed (changes requested):
> streams/al.m3u
>  24    error    "A2CNN.al" is on the blocklist due to claims of copyright holders (https://github.com/iptv-org/iptv/issues/16839)

https://github.com/iptv-org/iptv/actions/runs/16581717100/job/46899077190?pr=25371

eurohunter committed:
Update al.m3u

Removed A2 CNN, although the link was taken directly from its website which is publicly streaming live with no restrictions.

eurohunter commented:
I just removed A2 CNN. Was this channel specifically targeted by DMCA? This is an affiliate channel of CNN in the Albanian language, and it is publicly streaming on their website without any restrictions.

freearhey commented:
> A2 CNN was part of CNN Intl, and it's owned by WBD.

https://github.com/iptv-org/database/issues/18422

eurohunter commented:
> > A2 CNN was part of CNN Intl, and it's owned by WBD.
> 
> [iptv-org/database#18422](https://github.com/iptv-org/database/issues/18422)

I believe this channel has been mistakenly categorized as part of WBD. While it has partnered with CNN, its official name is A2 News (or simply A2) and is in fact owned 100% by the Albanian company Gener 2. I am providing some links that show the partnership and ownership of A2 by Gener 2, and not Warner Bros Discovery.
https://albania.mom-gmr.org/en/media/detail/outlet/a2-1/ 
https://www.broadbandtvnews.com/2018/02/07/new-cnn-channel-for-albania/
https://gener2.al/who-we-are/

This is why this channel was not part of the list shown in the DMCA list. Perhaps, to avoid confusion in the future, I can update the name to A2 News (not A2CNN)

eurohunter commented on streams/al.m3u:
```diff
@@ -1,55 +1,23 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlbKanaleMusicTV.al",AlbKanale Music TV (720p)
-https://albportal.net/albkanalemusic.m3u8
```
Thank you. This link does work indeed. After trying it with VPN now, I realized it was my ISP that was blocking the link.

eurohunter commented on streams/al.m3u:
```diff
@@ -1,55 +1,23 @@
-https://cdn.jwplayer.com/live/events/r2qgHu7W.m3u8
+https://tringliveviz.akamaized.net/delta/105/out/u/qwaszxerdfcvrtryuy.m3u8
+#EXTINF:-1 tvg-id="ZjarrTV.al",Zjarr TV (360p)
+https://zjarr.future.al/hls/playlist.m3u8
```
This one perhaps is not 24/7 because the live stream seems to be broken even on the website: https://zjarr.tv/live/
Yesterday it was working just fine.

BellezaEmporium replied:
Can confirm the link is currently working.

BellezaEmporium committed:
Merge branch 'master' into patch-1

BellezaEmporium committed:
reinstating over check

freearhey committed:
Update al.m3u
